packages/contracts/src/eip712/getVertexEIP712Values.ts

```diff
@@ -160,15 +160,15 @@ function getIsolatedOrderValues(
 ): EIP712IsolatedOrderValues {
   return {
     ...getOrderValues(params),
-    margin: BigInt(params.margin).toString(),
+    margin: toBigDecimal(params.margin).toFixed(0),
   };
 }
 
 function getListTriggerOrdersValues(
   params: EIP712ListTriggerOrdersParams,
 ): EIP712ListTriggerOrdersValues {
   return {
-    recvTime: BigInt(params.recvTime).toString(),
+    recvTime: params.recvTime,
```

Comment:
How come this doesn't need string conversion anymore, when it's a `BigDecimalish`?